src/azure-cli/azure/cli/command_modules/serviceconnector/_params.py

```diff
@@ -132,6 +132,13 @@ def add_new_addon_argument(context, source, target):
     def add_secret_store_argument(context):
         context.argument('key_vault_id', options_list=['--vault-id'], help='The id of key vault to store secret value')
 
+    def add_mysql_umi_argument(context, target):
+        if target in [RESOURCE.MysqlFlexible]:
+            context.argument('mysql_identity_id', options_list=['--mysql-identity-id'],
```

Comment:
why introducing an additional argument instead of resuing those in `AUTH_TYPE_PARAMS`.
and i noticed you added `AUTH_TYPE.SystemIdentity` for postgresq

Reply:
Good point